manifests/init.pp

```diff
@@ -321,6 +322,10 @@
     $sshd_config_subsystem_sftp_real = $sshd_config_subsystem_sftp
   }
 
+  if $sshd_config_subsystem_sftp_options != undef {
```

Comment:
Spec added here.

Reply:
That's the validation piece. Check out this commit to see how spec tests were added.

https://github.com/ghoneycutt/puppet-module-ssh/commit/bc4c8a82a64c41c2d146b038cbc3e8d6a4fc932b